class/Paym.js

```diff
@@ -12,9 +12,9 @@ export class Paym {
     this._isPaid = null;
   }
 
-  static get fee() {
-    return 0.003;
-  }
+  // static get fee() {
+  //   return 0.003;
+  // }
```

Comment:
Should remove code instead of commenting it out

Reply:
Please do ...